### DISCUSSION TITLE
Disabling format on save in a session

### DISCUSSION THREAD

**Discussion by brainwo** [neoclide/coc.nvim#4770] [Q&A]
Some projects I've been working on (that I only contributed for small feature) don't use a default formatter settings or provide any configuration files. This becomes so painful to work with as I need to keep turning on and off the format-on-save feature in the editor-wide configuration file or just disable coc with `:CocDisable` - which is also not ideal since I'm losing all the LSP feature along with it.

Is there any ways I could do to disable format-on-save for only in the current session?

If not, I'm suggesting command like `:CocDisableFormatting` or "disable formatting" option on `:CocCommand`.

Thanks in advance.

**Comment by fannheyward** [comment]
`:noa w`.

**Reply by brainwo** [reply]
That is for each save, how can I do this for one session?
(until I close the Neovim instance)
